src/serviceorchestration/5_0_0/sysd/serviceorchestration_sysd.tex

```diff
@@ -185,9 +185,9 @@ \subsection{Important Delimitations}
 \label{sec:delimitations}
 
 \begin{itemize}
-    \item There are three orchestration strategies (for now). A Service Orchestration implementation must support only one of those (but it may support multiple strategies). The administrator of the Local Cloud can decide what kind of orchestration strategy is used in the cloud, a consumer can not.
-    \item If the Local Cloud does not contain an Authentication system, there is no way for the Service Orchestration to verify the requester system. In that case, the Service Orchestration will consider the authentication data comes from the requester as valid.
-    \item If the Local Cloud does not contain a Service Registry, then Service Orchestration system with simple store strategy can be used. The two other strategies need the continuous support of the Service Registry system to provide their services.
+    \item There are three orchestration strategies (for now). A ServiceOrchestration implementation must support only one of those (but it may support multiple strategies). The administrator of the Local Cloud can decide what kind of orchestration strategy is used in the cloud, a consumer can not.
+    \item If the Local Cloud does not contain an Authentication system, there is no way for the ServiceOrchestration to verify the requester system. In that case, the ServiceOrchestration will consider the authentication data comes from the requester as valid.
```

Comment:
missing 'or does not using X.509 certificates' after Authentication System

ServiceOrchestration => \<Strategy\>ServiceOrchestration